dev-lang/mono/mono-4.4.1.0.ebuild

```diff
@@ -85,31 +90,34 @@ src_prepare() {
 
 	default_src_prepare
 	#eapply_user
+	multilib_copy_sources
 }
 
-src_configure() {
+multilib_src_configure() {
+	local big_arr="enable"
+#	ECONF_SOURCE="${S}"
+
+	multilib_is_native_abi || big_arr="disable"
+
 	local myeconfargs=(
 		--disable-silent-rules
 		$(use_with xen xen_opt)
 		--without-ikvm-native
 		--disable-dtrace
 		$(use_with doc mcs-docs)
 		$(use_enable nls)
+		--${big_arr}-big-arrays
```

Comment:
Err, wtf? Are you doing some poor assumptions at what native and non-native ABI will be? :-P